sdk/docs/manually-written/sdk/tutorials/setup-assistant/installation.rst

```diff
@@ -136,6 +136,28 @@ Download Manually
 
 If you want to verify the SDK download for security purposes before installing, you can look at :doc:`our detailed instructions for manual download and installation <manual-download>`.
 
+Troubleshooting
+***************
+
+The Daml installation requires x amount of disk space. If there is not enough space on disk, the installer outputs the error:
+``
+resource exhausted (No space left on device)
+``
+
+In order to fix this problem, one needs to ensure sufficient disk space overall, and in particuloar that the user's home directory and the temporary
+files directory have sufficient space (i.e., in the order of 2GiB each):
+
+* On Windows, one can find the user's home directory in the `%USERPROFILE%` environment variable, and the temporary files directory in the
+  ``%TEMP%`` environment variable
+* On Linux and MacOS, one can find the user's home directory in the ``HOME%`` environment variable, and the temporary files directory in the
+  ``TMPDIR`` environment variable 
+
+If the installer cannot ensure sufficient space in the the user's home directory and the temporary files directory, an alternative is to change
+the location of these directories before the installation.
+To do so, please set the ``DAML_HOME`` and ``TMPDIR`` environment variables to other directories that have sufficient space. You can find instructions on how to do this :doc:`here <path-variables>`.
```

Comment:
please avoid "here" references as per style-guide: https://docs.google.com/document/d/153D6-gI0blsAvlrzReHVyz8RCHUQzzdKnxnpcADlt6c/edit?tab=t.0#bookmark=id.ya2a6pwloo5b. Also a `:ref:` is preferred, as it is more stable under doc refactorings than a `:doc:` reference.